code/modules/mob/living/basic/space_fauna/spider/spider_abilities/hivemind.dm

```diff
@@ -51,7 +51,7 @@
 	return TRUE
 
 /**
- * Sends a message to all spiders from the target.
+ * Sends a big message to all spiders from the target.
```

Comment:
You forgot to remove the "big" part when you refactored the ability, it's not true any more.

Reply:
this one is for spider command which is still a big message